### PR TITLE
Hurd get network facts method extraction FIXME

### PR DESCRIPTION
**Pull request by Valesper**
##### SUMMARY
As part of a course project, I was directed to do basic refactoring on an open source project. The change I made was to address the FIXME in hurd.py, and extract the if block into a new method.

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
hurd.py

### CODE REVIEW

File: lib/ansible/module_utils/facts/network/hurd.py [hunk 1]
```diff
@@ -44,41 +44,43 @@ def populate(self, collected_facts=None):
...
+                            network_facts[current_if]['ipv4']['address'] = v
+                        elif k == 'netmask':
+                            network_facts[current_if]['ipv4']['netmask'] = v
+                        elif k == 'address6':
+                            address, prefix = v.split('/')
+                            network_facts[current_if]['ipv6'].append({
+                                'address': address,
+                                'prefix': prefix,
+                            })
+
```
Comment (ChrisAHolland) [inline lib/ansible/module_utils/facts/network/hurd.py]:
```suggestion

```

File: lib/ansible/module_utils/facts/network/hurd.py [hunk 2]
```diff
@@ -44,41 +44,43 @@ def populate(self, collected_facts=None):
...
-                        network_facts[current_if]['ipv4']['netmask'] = v
-                    elif k == 'address6':
-                        address, prefix = v.split('/')
-                        network_facts[current_if]['ipv6'].append({
-                            'address': address,
-                            'prefix': prefix,
-                        })
+        return get_network_facts(network_facts, fsysopts_path, socket_path)
 
-        return network_facts
```
Comment (ChrisAHolland) [inline lib/ansible/module_utils/facts/network/hurd.py:83]:
```suggestion
```

**Review by ChrisAHolland — CHANGES_REQUESTED** [review]
I think just these whitespace fixes plus a changelog and it'll pass

**Comment by sivel** [comment]
Thank you very much for your interest in Ansible.

Unfortunately, we do not accept pull requests that only achieve the goal of refactoring, unless it has been pre-approved by the core team.

Refactoring should only be performed when you are addressing a bug or feature that necessitates a refactoring of the immediate code.

If you have further questions please stop by IRC or the mailing list:

   * IRC: #ansible on irc.freenode.net
   * mailing list: https://groups.google.com/forum/#!forum/ansible-project

**Pull request closed without merging**